citi-onboarding/virtu-onepage
# Hotfix/icon and decoration line

### What I did
- Added the Virtù icon on page
- Added decoration lines (The one in the left is supposed to be on the Linkedin Section, but since it didn't exist yet i putted on the testimonials, and then when we merge the Linkedin branch, i'll change it here)

### How to test
1. Enter server folder with `cd server`
2. Execute the command `yarn` to download the server modules
3. Execute the command `yarn start` to start the server
4. Execute the command `cd ../client` and then `yarn` to download the client modules
5. Execute the command `yarn start` to start the client
6. Open your browser on `http://localhost:3001/admin` to view keystone admin
7. Log in with the credentials available at Notion
8. Open another browser page on `http://localhost:3000` to see the client page
9. See if the things I did are working
